.github/workflows/image-factory.yaml

```diff
@@ -126,7 +133,7 @@ jobs:
           KO_DOCKER_REPO=quay.io/securesign KO_DEFAULTBASEIMAGE=registry.access.redhat.com/ubi9/ubi-micro make ko-resolve
 
   build-tuf:
-    runs-on: ubuntu-20.04
+    runs-on: $RUNS_ON
```

Comment:
this would need to env and then var but I think we just hardcode the OS in. I don't know if the templating of vars works at this level of job defintion